CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.1.0 (Unreleased)
+
+## Enhancements
+
+* Add beta support for SSOTeamID to `Team`, `TeamCreateOptions`, `TeamUpdateOptions` by @xlgmokha [#364](https://github.com/hashicorp/go-tfe/pull/364)
```

Comment:
@radditude Is it okay to say `beta` here?

Reply:
Yeah, that seems like a good way to flag it so that TFE users will have a heads up that it's not available in TFE just yet!